lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.spec.ts

```diff
@@ -73,6 +73,18 @@ describe('TaskHeaderCloudComponent', () => {
             component.ngOnChanges();
         });
 
+        it('should hide the title if showTitle is false', () => {
+            component.showTitle = false;
+            fixture.detectChanges();
+            let taskTitle = fixture.debugElement.query(By.css('.adf-task-title'));
+            expect(taskTitle).toBeNull();
+
+            component.showTitle = true;
+            fixture.detectChanges();
+            taskTitle = fixture.debugElement.query(By.css('.adf-task-title'));
+            expect(taskTitle).not.toBeNull();
```

Comment:
can we avoid toBeNull kind of check they are still ok if is undefined

Reply:
I added another check because `expect(null).toBeDefined()` will return true